clarity/src/vm/errors.rs

```diff
@@ -126,28 +126,28 @@ pub enum WasmError {
     TopLevelNotFound,
     MemoryNotFound,
     GlobalNotFound(String),
-    #[cfg(feature = "canonical")]
+    #[cfg(feature = "clarity_wasm")]
```

Comment:
Should this entire `WasmError` be behind the feature flag?

Reply:
Good question. Probably yes. I don't know a place, outside `clarity-wasm`, where WasmError is used.

Reply:
Good call.
I refactored it a little.
Also I had to force push, I realized that this had the wrong commit (I locally renamed `clarity_wasm` into `clarity-wasm` – matching `developer-mode`), but opened a PR with a wrong version